packages/block-editor/src/components/plain-text/index.native.js

```diff
@@ -7,7 +7,8 @@ import { TextInput, Platform, Dimensions } from 'react-native';
  * WordPress dependencies
  */
 import { Component } from '@wordpress/element';
-import { RichText, getPxFromCssUnit } from '@wordpress/block-editor';
+import { RichText } from '@wordpress/block-editor';
```

Comment:
Shouldn't this just be a local import? You have some more similar imports below.

Reply:
Yes, though there's a lot more of these. Could address this one though.